crates/core/src/node/inner/in_memory_inner.rs

```diff
@@ -947,7 +964,9 @@ impl InMemoryNodeInner {
                     message
                 );
 
```

Comment:
Would  it be  possible to pass the error up the execution stack and handle output uniformly in a single location?

Also may be done in a separate PR.

Reply:
I will defer to subsequent PR given the extent of this PR